beebop/utils.py

```diff
@@ -14,21 +14,29 @@
 
 
 def get_args():
+    """
+    Reads in arguments from file
```

Comment:
For docs like this, try not to describe what it is doing (which is easy enough to read here) but *why* it's doing it. This could  be something like "Read in fixed arguments to poppunk that are always set, or used as defaults. This is needed because of the large number of arguments that poppunk needs". Or something like that